modules/mobile/app/controllers/mobile/v0/contact_info_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class ContactInfoController < ApplicationController
+      before_action :map_logingov_to_idme
+      after_action :handle_vet360_id, only: :show
+
+      def show
+        render json: Mobile::V0::ContactInfoSerializer.new(@current_user, @current_user.vet360_contact_info)
+      end
+
+      private
+
+      # solution so old app versions will still treat LOGINGOV accounts as multifactor
+      def map_logingov_to_idme
```

Comment:
Hmm. This is interesting and I hadn't thought about it. Jayson added this to the users controller. It doesn't seem like it impacts anything here. I suspect it was only being done because the user serializer includes `signin_service`, in which case it wouldn't be needed here. Maybe @jperk51 can provide insight.

Reply:
I'll wait for @jperk51 to give some feedback before removing it

Reply:
From reading the code, I'm pretty sure that was added because the app was doing some sort of check for 'oauth_IDME' so it shouldn't be necessary for this endpoint. If the FE has issues with it being gone then the FE needs to be updated as part of integrating with this endpoint.